README.md

```diff
@@ -106,10 +106,16 @@ In the case of the `--force` option has the following effects:
 * Idle state of workers will be ignored
 * uptime/billing hour usage of workers will be ignored
 
-### --dry-run optionn
+### --dry-run option
 
 If `--dry-run` is enabled the process will run through and report what it would potentially do, but no instances will be started, stopped or taken in/out of maintenance.
 
+### --scale-available option
+
+By default the `scale up` and `scale to` commands will fail with a message about "not enough workers" if asked to scale
+a cluster beyond the available number of workers. With the `--scale-available` flag the process will still warn about
+the condition but continue to scale up the available number of instances.
+

```

Comment:
this is a little confusing to me. In the cron jobs, we ask for 10 workers and this fails if we're running 2 and a third broken one. This implies that the cron job would be asking to scale up 8 workers.